src/components/home/common/hamburgerMenu/hamburgerMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import 'components/home/common/hamburgerMenu/hamburgerMenu.css'
+import menu from 'assets/images/menu.png' 
+import { useState } from 'react';
+
+const HamburguerMenu = () => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const [showAboutLink, setShowAboutLink] = useState(false);
+
+  const toggleMenu = () => {
+    setMenuOpen(!isMenuOpen);
```

Comment:
It's good practice to use the previous state value you get from the callback, rather that using manually the value
```suggestion
    setMenuOpen(prevState => !prevState);
```